### DISCUSSION TITLE
AMP tutorial does not work on 1.5.0

### DISCUSSION THREAD

**Discussion by blchu** [apache/incubator-mxnet#15866] [Q&A] [answered]
The AMP tutorial offered in the iPython notebook at https://mxnet.incubator.apache.org/versions/master/tutorials/amp/amp_tutorial.html results on errors in certain function calls when running with 1.5.0, notably `test_utils.download_model` and `contrib.amp.convert_hybrid_block`. Peeking at the source code, I don't think these functions are implemented in 1.5.0. There appears to be no version requirement specified for AMP or the tutorial, and attempting to change the version from the dropdown does not change anything.

[a comment by an automated account is not shown]

**Comment by anirudh2290** [comment, marked as answer]
@bichu thanks for bringing this up. You are right, convert_hybrid_block isn't available in 1.5 but its in master. you can obtain it with: 

```
pip install --pre mxnet-cu100mkl
```

About the documentation, there was a dropdown in mxnet.io to select documentation for different versions, but it doesnt seem to be working. cc @aaronmarkham for the version selector dropdown issue.

**Comment by NRauschmayr** [comment]
Thanks for reporting this. We are actively working on website changes and documentation, where the version dropdown mechanism will be modified. We will make sure that this issue will be fixed.

**Comment by aaronmarkham** [comment]
You can disable the redirect for tutorials by modifying .htaccess.
https://github.com/apache/incubator-mxnet/blob/master/docs/build_version_doc/artifacts/.htaccess#L10
None of the tutorials in branches other than master are tested, so this change would introduce the issue that people can visit broken tutorials. 
Tutorials really should have a prerequisites section that defines the minimum version(s) required, so I think a better change would be to update the tutorial to tell the user what they need.
